src/presentation-markup.html

```diff
@@ -4152,28 +4156,7 @@ <h5 id="presm_mstyle_attrs">Attributes</h5>
    see <a href="#presm_lbattrs"></a>
   </td>
  </tr>
- 
- <tr>
- <td rowspan="2" class="attname"><span class="coreno">decimalpoint</span></td>
```

Comment:
was this deletion intentended in this PR?
If we delete it from the mstyle-attributes don't we need to add it as a common presentation  mathml, or or mrow attribute, as it's used elsewhere (also the schema would need an update to match)